perftestnotebook.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
This is fine, but not what I had in mind for a CLI tool. Here's an example of what you need to do: https://github.com/gmierz/mindwave-lsl/blob/master/setup.py

The entry points is where you make the command `perftestnotebook` and have it run the main function in perftestnotebook.py.

You will have to move all of the actual code into a folder called `perftestnotebook` as well. There will be many other changes to existing code that will need to be done as well.